app/views/layouts/mailer.html.erb

```diff
@@ -126,28 +326,27 @@
               </tr>
               </tbody>
             </table>
-            <table class="footer" style="box-sizing: border-box; margin-top: 50px; color: rgb(152, 156, 165); text-align: center; font-size: 11px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px;" align="center">
-              <tbody style="box-sizing: border-box;">
-              <tr style="box-sizing: border-box;">
-                <td class="footer-cell" style="box-sizing: border-box;">
-                </td>
+            <table class="footer">
+              <tbody>
+              <tr>
+                <td class="footer-cell"></td>
               </tr>
               </tbody>
             </table>
-            <div class="c2577" style="text-align:center;box-sizing: border-box; padding-top: 10px; padding-right: 10px; padding-bottom: 10px; padding-left: 10px;">
-              <p class="footer-info" style="box-sizing: border-box;">Court Appointed Special Advocate (CASA) /
+            <div class="c2577">
+              <p class="footer-info">Court Appointed Special Advocate (CASA) /
                 <%= @casa_organization.name %>
-                <br style="box-sizing: border-box;">
-                <br style="box-sizing: border-box;"><%= @casa_organization.address %>
+                <br>
+                <br><%= @casa_organization.address %>
               </p>
-              <table id="iwvod" style="box-sizing: border-box; height: 150px; margin: 0 auto 10px auto; padding: 5px 5px 5px 5px; width: 100%;" width="100%" height="150">
-                <tbody style="box-sizing: border-box;">
-                <tr style="box-sizing: border-box;">
-                  <td id="igyvi" style="box-sizing: border-box; font-size: 12px; font-weight: 300; vertical-align: top; color: rgb(111, 119, 125); margin: 0; padding: 0; width: 50%;" width="50%" valign="top">
-                    <a id="izi4m" style="box-sizing: border-box; color: #000;">About CASA</a>
+              <table id="iwvod">
+                <tbody>
+                <tr>
+                  <td id="igyvi">
+                    <a id="izi4m">About CASA</a>
                   </td>
-                  <td id="idvsy" style="box-sizing: border-box; font-size: 12px; font-weight: 300; vertical-align: top; color: rgb(111, 119, 125); margin: 0; padding: 0; width: 50%;" width="50%" valign="top">
-                    <a id="ilo0w" style="box-sizing: border-box; color: #000;">Update Profile</a>
+                  <td id="idvsy">
+                    <a id="ilo0w">Update Profile</a>
```

Comment:
The two links on lines 349 and 346 are not pointing to anything. Should this be fixed?

Reply:
Yes, the links should be clickable. Line 349 says Update Profile so it should point to the update profile path. Thanks for noticing! :)